pkg/module_manager/models/modules/basic.go

```diff
@@ -63,10 +65,13 @@ func NewBasicModule(name, path string, order uint32, staticValues utils.Values,
 		return nil, fmt.Errorf("new values storage: %w", err)
 	}
 
+	crdFilesPaths, _ := filepath.Glob(filepath.Join(path, "crds", "*.yaml"))
```

Comment:
Let's exclude `doc-` files here and also exclude files started with `_`

Reply:
DONE